packages/office-ui-fabric-react/src/components/Facepile/Facepile.Props.ts

```diff
@@ -76,6 +76,11 @@ export interface IFacepilePersona extends React.HTMLProps<HTMLButtonElement | HT
   onMouseOut?: (ev?: React.MouseEvent<HTMLElement>, persona?: IFacepilePersona) => void;
 
   /**
+   * If provided, the handler will be called when the component is mounted or unmounted
+   */
+  refHandler?: (el?: HTMLElement, persona?: IFacepilePersona) => void;
+
```

Comment:
Indeed, that works, must have done something weird before.
It gives me a Persona object which is a React component, and I need to get the corresponding HTMLElement.
ReactDOM.findDOMNode does the trick so I'll probably continue with that without needing to do any changes in Fabric.

Thanks for the suggestion and stopping me from duplicating stuff!